flink-connectors/flink-connector-kafka-0.11/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducer011ITCase.java

```diff
@@ -592,24 +595,21 @@ private Properties createProperties() {
 		return properties;
 	}
 
-	private void assertIsCausedBy(Class<?> clazz, Throwable ex) {
+	private static <T> Optional<T> isCausedBy(Class<T> clazz, Throwable ex) {
```

Comment:
Better to use `ExceptionUtils#findThrowable`.

Reply:
Changed to `findThrowable` :)